plugins/inputs/mysql/v2/convert.go

```diff
@@ -70,12 +81,27 @@ func ParseValue(value sql.RawBytes) (interface{}, error) {
 }
 
 var GlobalStatusConversions = map[string]ConversionFunc{
-	"ssl_ctx_verify_depth": ParseInt,
-	"ssl_verify_depth":     ParseInt,
+	"innodb_available_undo_logs":    ParseUint,
+	"innodb_buffer_pool_pages_misc": ParseUint,
+	"innodb_data_pending_fsyncs":    ParseUint,
+	"ssl_ctx_verify_depth":          ParseUint,
+	"ssl_verify_depth":              ParseUint,
 }
 
 var GlobalVariableConversions = map[string]ConversionFunc{
-	"gtid_mode": ParseGTIDMode,
+	"delay_key_write":                  ParseString,        // ON, OFF, ALL
+	"enforce_gtid_consistency":         ParseString,        // ON, OFF, WARN
+	"event_scheduler":                  ParseBoolAsInteger, // YES, NO, DISABLED
+	"gtid_mode":                        ParseGTIDMode,
+	"have_openssl":                     ParseBoolAsInteger, // alias for have_ssl
+	"have_ssl":                         ParseBoolAsInteger, // YES, DISABLED
+	"have_symlink":                     ParseBoolAsInteger, // YES, NO, DISABLED
+	"session_track_gtids":              ParseString,
+	"session_track_transaction_info":   ParseString,
+	"slave_skip_errors":                ParseString,
+	"ssl_fips_mode":                    ParseString,
+	"transaction_write_set_extraction": ParseString,
+	"use_secondary_engine":             ParseString,
```

Comment:
Are there other variables that may need to be parsed as string? Did you add only the variables here that we know are causing trouble or were you able to find a list of vars and expected values in mysql docs or code and make an exhaustive list? The current list of fixes is great, I just wonder if there are more out there that need to be fixed.

Reply:
Hey @reimda, I took variables from https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html. I guess there is no simple list of string variables.

We can extend our mapping with more variables at any time.